muxnect/server.py

```diff
@@ -110,9 +109,9 @@ def command_line():
     bind_address = args.bind_address
     port = args.port
 
-    socket_code =  port_is_busy(port)
+    socket_code = port_is_busy(port)
     if socket_code:
-        raise OSError('[Errno {}] Cannot start web server on port {}'.format(socket_code, port))
+        raise OSError('[Errno {0}] Cannot start web server on port {}'.format(socket_code, port))
```

Comment:
Just for consistency, it works without the index but you use them at some other ``print``s, even though it's more verbose it's explicit.

Reply:
Yep, good idea. Although, we should set index for port to `{1}` as well.